plugin/core/sessions.py

```diff
@@ -442,7 +444,7 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
         capabilities['experimental'] = cast(LSPObject, config.experimental_capabilities)
     if get_file_watcher_implementation():
         workspace_capabilites["didChangeWatchedFiles"] = {"dynamicRegistration": True}
-    return {
+    return cast(InitializeParams, {
```

Comment:
We could do better by casting `initializationOptions` to `LSPAny` and `workspaceFolders` to `WorkspaceFolders`. Although there is a conflict with our `WorkspaceFolders` type so not sure how much work would it be.